tools/validators/instance_validator/generate_universe.py

```diff
@@ -0,0 +1,63 @@
+import sys
+import os
+
+def build_universe():
+    """Generates ontology universe.
+
+    Returns:
+        Generated universe object.
+    """
+    # add universe building packages to path
+    sys.path.append(os.path.abspath(os.path.join('..', 'ontology_validator'))) 
+    # add ontology files to path
+    sys.path.append(os.path.abspath(os.path.join('../../../', 'ontology')))
+
+    from yamlformat.validator import external_file_lib
```

Comment:
move this up to the begining to the file with the import best practices from the early PR

Reply:
Done